### PR TITLE
models: add start and finish times for jobs

### PR DESCRIPTION
**Pull request by diegodelemos**
Closes reanahub/reana-client#497.

### CODE REVIEW

**Review by ParthS007 — APPROVED** [review]
Thanks for the PR. Tested locally, working fine 🚀

File: reana_db/models.py [hunk 1]
```diff
@@ -656,10 +656,28 @@ class Job(Base, Timestamp):
...
+    if new_status != old_status:
+        if new_status in [
+            JobStatus.finished,
+            JobStatus.failed,
+        ]:
+            job.finished_at = datetime.now()
+        elif new_status in [JobStatus.running]:
+            job.started_at = datetime.now()
+
+    Session.commit()
```
Comment (mvidalgarcia) [inline reana_db/models.py:678]: Should we commit in the _very rare_ case that `new_status` == `old_status`? Maybe this commit should only happen when they're different, what do you think?

**Review by mvidalgarcia — APPROVED** [review]
Haven't tested it. Code looks good 👍

Just a minor comment.

File: reana_db/models.py [hunk 2]
```diff
@@ -663,10 +663,29 @@ class Job(Base, Timestamp):
     env_vars = Column(JSONType)
     deleted = Column(Boolean)
     logs = Column(String, nullable=True)
+    started_at = Column(DateTime)
+    finished_at = Column(DateTime)
```
Comment (tiborsimko) [inline reana_db/models.py:667]: Do we have Alembic upgrade recipe for this change? Or shall we compile them afterwards before releasing?

Comment (mvidalgarcia) [reply reana_db/models.py:667]: Definitely needed.

**Pull request merged**